assets/js/main.js

```diff
@@ -0,0 +1,18 @@
+//Global App name is ProRubric
+var appModule = angular.module('ProRubric', []);
+
+//Required to remove the conviction between Handlebars and Angular.
+appModule.config(function($interpolateProvider) {
+    $interpolateProvider.startSymbol('{[{');
+    $interpolateProvider.endSymbol('}]}');
+});
+
+appModule.controller('mainController', function($scope) {
+
+    $scope.addDegree = function(degreeName, degreeAcronym){
+        console.log('Degree Name: ', degreeName);
+        console.log('Degree Acronym: ', degreeAcronym);
+    };
+
+
+});
```

Comment:
Corretions for **assets/js/main.js**
```javascript
//Global App name is ProRubric
angular.module('ProRubric', []);

//Required to remove the conviction between Handlebars and Angular.
angular.module('ProRubric')
.config(function($interpolateProvider) {
    $interpolateProvider.startSymbol('{[{');
    $interpolateProvider.endSymbol('}]}');
});

angular.module('ProRubric')
.controller('mainController', function($scope) {
    $scope.addDegree = function(degree){
        //pass in one object with all the properties attached to that object
        console.log('Degree Name: ', degree.name);
        console.log('Degree Acronym: ', degree.acronym);
    };
});
```

1. Just use the angular.module() getter instead of setting the whole module to a variable. 
2. Also to get around magnification we'll need to include ng-annotate to handle Angular's dependency injection.  
 * gulp-ng-annotate